app/src/userpages/components/StreamPage/Show/InfoView/index.jsx

```diff
@@ -109,6 +109,7 @@ export const InfoView = ({ disabled }: Props) => {
                     {I18n.t('userpages.streams.edit.details.streamId')}
                 </Label>
                 <ActionsDropdown
+                    disabled={disabled}
```

Comment:
@mondoreale Hm, read-only users should still be able to copy the stream id

Reply:
Oh, damn, yeah, I guess. I went full-disable with this one accidently. 😱